# Galician translation of libgda.
# Copyright (C) 2000 Jesús Bravo Álvarez.
# Jesús Bravo Álvarez <jba@pobox.com>, 2000.
#
# Proxecto Trasno - Adaptación do software libre á lingua galega:  Se desexas
# colaborar connosco, podes atopar máis información en http://trasno.gpul.org
#
# First Version: 2000-04-29 12:07+0200
#
msgid ""
msgstr ""
"Project-Id-Version: libgda\n"
"POT-Creation-Date: 2001-01-22 18:36+0100\n"
"PO-Revision-Date: 2000-10-29 15:03+0100\n"
"Last-Translator: Jesús Bravo Álvarez <jba@pobox.com>\n"
"Language-Team: Galician <trasno@ceu.fi.udc.es>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=iso-8859-1\n"
"Content-Transfer-Encoding: 8bit\n"

#: lib/gda-client/gda-connection-pool.c:267
#, c-format
msgid "could not open connection to %s"
msgstr "non foi posible abrir unha conexión a %s"

#: lib/gda-client/gda-connection-pool.c:280
#, c-format
msgid "Data source %s not found"
msgstr "Fonte de datos %s non atopada"

#: lib/gda-client/gda-connection.c:234 lib/gda-client/gda-error.c:179
#, c-format
msgid "%s: The server didn't respond."
msgstr "%s: O servidor non respondeu."

#: lib/gda-client/gda-connection.c:238 lib/gda-client/gda-error.c:183
#, c-format
msgid "%s: An Error occured in the CORBA system."
msgstr "%s: Ocorreu un erro no sistema CORBA."

#: lib/gda-client/gda-connection.c:493
#, c-format
msgid "Database '%s' not found in system configuration"
msgstr "Base de datos '%s' non atopada na configuración do sistema"

#: lib/gda-client/gda-connection.c:509
msgid "Could not open CORBA factory"
msgstr "Non foi posible abri-la fábrica CORBA"

#: lib/gda-client/gda-field.c:245
msgid "<NULL>"
msgstr "<NULL>"

#: lib/gda-client/gda-field.c:251 lib/gda-client/gda-field.c:253
msgid "<Unknown GDA Type(NULL)>"
msgstr "<Tipo GDA Descoñecido (NULL)>"

#: lib/gda-client/gda-field.c:266
msgid "TRUE"
msgstr "CERTO"

#: lib/gda-client/gda-field.c:267
msgid "FALSE"
msgstr "FALSO"

#: lib/gda-common/gda-corba.c:73
#, c-format
msgid "CORBA System exception: %s"
msgstr "Excepción de Sistema de CORBA: %s"

#: lib/gda-common/gda-corba.c:102
msgid "True"
msgstr "Certo"

#: lib/gda-common/gda-corba.c:102
msgid "False"
msgstr "Falso"

#: lib/gda-common/gda-init.c:59
msgid "Attempt to initialize an already initialized client"
msgstr "Tentouse inicializar un cliente xa inicializado"

#: lib/gda-common/gda-log.c:48
msgid "Saving log...x"
msgstr "Gardando rexistro..."

#. compose message
#: lib/gda-common/gda-log.c:69
msgid "ERROR: "
msgstr "ERRO: "

#: lib/gda-common/gda-log.c:69
msgid "MESSAGE: "
msgstr "MENSAXE: "

#: lib/gda-common/gda-thread.c:211
msgid "thread is already running"
msgstr "o fío aínda está en execución"

#: lib/gda-common/gda-xml-database.c:282
#, c-format
msgid "table %s already exists"
msgstr "a táboa %s xa existe"

#: lib/gda-common/gda-xml-database.c:446
#, c-format
msgid "%p is not a valid table"
msgstr "%p non é unha táboa válida"

#: lib/gda-server/gda-server-error.c:185
msgid "Could not get pointer to server implementation"
msgstr "Non se pode obte-lo punteiro á implementación do servidor"

#: lib/gda-server/gda-server-impl-connection.c:123
#: lib/gda-server/gda-server-impl-connection.c:132
#: lib/gda-server/gda-server-impl-connection.c:139
#: lib/gda-server/gda-server-impl-connection.c:148
#: lib/gda-server/gda-server-impl-connection.c:155
#: lib/gda-server/gda-server-impl-connection.c:164
#: lib/gda-server/gda-server-impl-connection.c:171
#: lib/gda-server/gda-server-impl-connection.c:180
#: lib/gda-server/gda-server-impl-recordset.c:126
#: lib/gda-server/gda-server-impl-recordset.c:135
#: lib/gda-server/gda-server-impl-recordset.c:142
#: lib/gda-server/gda-server-impl-recordset.c:157
#: lib/gda-server/gda-server-impl-recordset.c:166
#: lib/gda-server/gda-server-impl-recordset.c:173
#: lib/gda-server/gda-server-impl-recordset.c:182
#: lib/gda-server/gda-server-impl-recordset.c:189
#: lib/gda-server/gda-server-impl-recordset.c:198
#: lib/gda-server/gda-server-impl-recordset.c:205
#: lib/gda-server/gda-server-impl-recordset.c:213
#: lib/gda-server/gda-server-impl-recordset.c:222
#: lib/gda-server/gda-server-impl-recordset.c:229
#: lib/gda-server/gda-server-impl-recordset.c:237
#: lib/gda-server/gda-server-impl-recordset.c:245
#: lib/gda-server/gda-server-impl-recordset.c:273
#: lib/gda-server/gda-server-impl-recordset.c:281
#: lib/gda-server/gda-server-impl-recordset.c:289
#: lib/gda-server/gda-server-impl-recordset.c:297
#: lib/gda-server/gda-server-impl-recordset.c:305
#: lib/gda-server/gda-server-impl-recordset.c:314
#, c-format
msgid "%s not implemented"
msgstr "%s non implementado"

#: lib/gda-server/gda-server-impl-connection.c:272
#, c-format
msgid "impl_GDA_Connection_open: opening connection with DSN: %s"
msgstr "impl_GDA_Connection_open: abrindo a conexión co DSN: %s"

#: lib/gda-server/gda-server-impl-recordset.c:407
#, c-format
msgid "%s: an error ocurred while fetching data"
msgstr "%s: ocorreu un erro recibindo os datos"

#: lib/gda-server/gda-server-impl-recordset.c:446
#, c-format
msgid "%s: length = %d"
msgstr "%s: lonxitude = %d"

#: lib/gda-server/gda-server-impl-recordset.c:459
#, c-format
msgid "%s: sending field description for '%s'"
msgstr "%s: enviando a descrición do campo '%s'"

#: lib/gda-server/gda-server-impl.c:184
#, c-format
msgid "Starting provider %s with no implementation functions"
msgstr "Iniciando o fornecedor %s sen funcións de implementación"

#: lib/gda-server/gda-server-impl.c:210
#: report/server/gda-report-server-main.c:113
msgid ""
"OAF doesn't know about our IID; indicates broken installation. Exiting..."
msgstr ""
"OAF descoñece o noso IID; isto indica unha instalación errónea. Saíndo..."

#: lib/gda-server/gda-server-impl.c:213
msgid ""
"Another instance of this provider is already registered with OAF. Exiting..."
msgstr "Xa hai rexistrada outra instancia deste fornecedor con OAF. Saíndo..."

#: lib/gda-server/gda-server-impl.c:216
msgid "Unknown error registering this provider with OAF. Exiting"
msgstr "Erro descoñecido rexistrando este fornecedor con OAF. Saíndo"

#: lib/gda-server/gda-server-impl.c:220
#, c-format
msgid "Registered with ID = %s"
msgstr "Rexistrado co ID = %s"

#: lib/gda-server/gda-server-impl.c:296
#, c-format
msgid "CORBA system exception %s"
msgstr "Excepción de sistema de CORBA %s"

#: lib/gda-server/gda-server-impl.c:299
#, c-format
msgid "CORBA user exception: %s"
msgstr "Excepción de usuario de CORBA: %s"

#: lib/gda-server/gda-server-init.c:63 lib/gda-server/gda-server-init.c:69
#, c-format
msgid "Received signal %d, shutting down"
msgstr "Recibiuse o sinal %d, saíndo"

#: lib/gda-server/gda-server-init.c:73
#, c-format
msgid "Received signal %d. Ignoring"
msgstr "Recibiuse o sinal %d. Ignorando"

#: lib/gda-server/gda-server-init.c:123
msgid "Attempt to initialize an already initialized provider"
msgstr "Tentouse inicializar un fornecedor xa inicializado"

#: providers/gda-interbase-server/gda-interbase-connection.c:261
#: providers/gda-ldap-server/gda-ldap-connection.c:272
#: providers/gda-mdb-server/gda-mdb-connection.c:360
#: providers/gda-mysql-server/gda-mysql-connection.c:387
#: providers/gda-odbc-server/gda-odbc-connection.c:609
#: providers/gda-odbc-server/gda-odbc-connection.c:642
#: providers/gda-oracle-server/gda-oracle-connection.c:426
#: providers/gda-postgres-server/gda-postgres-connection.c:779
#: providers/gda-sybase-server/gda-sybase-error.c:109
#, c-format
msgid "error '%s' at %s"
msgstr "erro '%s' en %s"

#: providers/gda-interbase-server/gda-interbase-connection.c:266
#: providers/gda-interbase-server/gda-interbase-connection.c:267
#: providers/gda-ldap-server/gda-ldap-connection.c:277
#: providers/gda-ldap-server/gda-ldap-connection.c:278
#: providers/gda-mdb-server/gda-mdb-connection.c:364
#: providers/gda-mdb-server/gda-mdb-connection.c:365
#: providers/gda-mysql-server/gda-mysql-connection.c:391
#: providers/gda-mysql-server/gda-mysql-connection.c:392
#: providers/gda-odbc-server/gda-odbc-connection.c:611
#: providers/gda-odbc-server/gda-odbc-connection.c:612
#: providers/gda-odbc-server/gda-odbc-connection.c:644
#: providers/gda-odbc-server/gda-odbc-connection.c:645
#: providers/gda-oracle-server/gda-oracle-connection.c:431
#: providers/gda-oracle-server/gda-oracle-connection.c:432
#: providers/gda-postgres-server/gda-postgres-connection.c:786
#: providers/gda-postgres-server/gda-postgres-connection.c:787
#: providers/gda-sybase-server/gda-sybase-error.c:118
#: providers/gda-sybase-server/gda-sybase-error.c:119
msgid "Not available"
msgstr "Non dispoñible"

#: providers/gda-interbase-server/gda-interbase-connection.c:268
#: providers/gda-ldap-server/gda-ldap-connection.c:279
#: providers/gda-mdb-server/gda-mdb-connection.c:366
#: providers/gda-mysql-server/gda-mysql-connection.c:393
#: providers/gda-oracle-server/gda-oracle-connection.c:433
#: providers/gda-postgres-server/gda-postgres-connection.c:788
#: providers/gda-sybase-server/gda-sybase-error.c:120
msgid "error"
msgstr "erro"

#: providers/gda-ldap-server/main-ldap.c:69
msgid "Could not register LDAP provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor LDAP"

#: providers/gda-interbase-server/gda-interbase-connection.c:260
#, c-format
msgid "Error code %ld"
msgstr "Código de erro %ld"

#: providers/gda-interbase-server/main-interbase.c:70
msgid "Could not register INTERBASE provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor INTERBASE"

#: providers/gda-mail-server/gda-mail-connection.c:108
#: providers/gda-mdb-server/gda-mdb-connection.c:224
#: providers/gda-mysql-server/gda-mysql-connection.c:251
#: providers/gda-oracle-server/gda-oracle-connection.c:295
#: providers/gda-postgres-server/gda-postgres-connection.c:511
#: providers/gda-sybase-server/gda-sybase-connection.c:269
#, c-format
msgid "Unhandled SCHEMA_QTYPE %d"
msgstr "SCHEMA_QTYPE %d non manexado"

#: providers/gda-mail-server/main-mail.c:70
msgid "Could not register Mail provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor de correo"

#. execute command
#: providers/gda-mdb-server/gda-mdb-command.c:86
#: providers/gda-mysql-server/gda-mysql-command.c:87
#: providers/gda-sybase-server/gda-sybase-command.c:120
#, c-format
msgid "executing command '%s'"
msgstr "executando o comando '%s'"

#: providers/gda-mdb-server/gda-mdb-command.c:88
#: providers/gda-mysql-server/gda-mysql-command.c:89
#, c-format
msgid "info '%s'"
msgstr "info '%s'"

#: providers/gda-mdb-server/gda-mdb-connection.c:137
#: providers/gda-mysql-server/gda-mysql-connection.c:148
#, c-format
msgid ""
"%s:%d: You cannot provide a UNIX_SOCKET if you also provide either a HOST or "
"a PORT. Please remove the UNIX_SOCKET, or remove both the HOST and PORT "
"options"
msgstr ""
"%s:%d: Non pode indicar un UNIX_SOCKET se tamén dá unha MÁQUINA ou un PORTO. "
"Quite o UNIX_SOCKET, ou as opcións MÁQUINA e PORTO"

#: providers/gda-mdb-server/gda-mdb-connection.c:147
#, c-format
msgid "%s:%d: You must  provide the filename with the MS Access database"
msgstr "%s:%d: Ten que indica-lo nome do ficheiro coa base de datos MS Access"

#: providers/gda-mdb-server/gda-mdb-connection.c:152
#, c-format
msgid "Opening connection with filename=%s, user=%s, password=%s"
msgstr "Abrindo a conexión con ficheiro=%s, usuario=%s, contrasinal=%s"

#: providers/gda-mdb-server/gda-mdb-connection.c:415
#: providers/gda-mysql-server/gda-mysql-connection.c:442
msgid "mysql_recset is NULL"
msgstr "mysql_recset é NULL"

#: providers/gda-mdb-server/gda-mdb-connection.c:457
#: providers/gda-mysql-server/gda-mysql-connection.c:484
#: providers/gda-sybase-server/gda-sybase-connection.c:432
msgid "No table name given for SCHEMA_COLS"
msgstr "Non se indicou un nome de táboa para SCHEMA_COLS"

#: providers/gda-mdb-server/main-mdb.c:70
#: providers/gda-mysql-server/main-mysql.c:70
msgid "Could not register MySQL provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor MySQL"

#: providers/gda-mysql-server/gda-mysql-connection.c:162
#, c-format
msgid ""
"Opening connection with user=%s, password=%s, host=%s, port=%s, "
"unix_socket=%s"
msgstr ""
"Abrindo a conexión con usuario=%s, contrasinal=%s, máquina=%s, porto=%s, "
"unix_socket=%s"

#: providers/gda-odbc-server/gda-odbc-connection.c:280
#, c-format
msgid "gda_odbc_connection_open_schema: Unhandled SCHEMA_QTYPE %d\n"
msgstr "gda_odbc_connection_open_schema: SCHEMA_QTYPE %d non manexado\n"

#: providers/gda-odbc-server/gda-odbc-connection.c:626
#: providers/gda-odbc-server/gda-odbc-connection.c:627
#: providers/gda-odbc-server/gda-odbc-connection.c:628
#: providers/gda-odbc-server/gda-odbc-connection.c:659
#: providers/gda-odbc-server/gda-odbc-connection.c:660
#: providers/gda-odbc-server/gda-odbc-connection.c:661
msgid "no text"
msgstr "sen texto"

#: providers/gda-odbc-server/main-odbc.c:69
msgid "Could not register ODBC provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor ODBC"

#: providers/gda-oracle-server/gda-oracle-command.c:185
msgid "Could not allocate statement handle"
msgstr "Non foi posible asignar memoria para o manexador da declaración"

#: providers/gda-oracle-server/gda-oracle-command.c:189
msgid "command object being created without associated connection"
msgstr "o obxecto de comando estase a crear sen unha conexión asociada"

#: providers/gda-oracle-server/gda-oracle-command.c:220
#, c-format
msgid "Executing command '%s'"
msgstr "Executando o comando '%s'"

#: providers/gda-oracle-server/gda-oracle-connection.c:113
msgid "error in OCIHandleAlloc() HTYPE_SVCCTX"
msgstr "erro en OCIHandleAlloc() HTYPE_SVCCTX"

#: providers/gda-oracle-server/gda-oracle-connection.c:116
msgid "error in OCIHandleAlloc() HTYPE_SERVER"
msgstr "erro en OCIHandleAlloc() HTYPE_SERVER"

#: providers/gda-oracle-server/gda-oracle-connection.c:119
msgid "error in OCIHandleAlloc() HTYPE_ERROR"
msgstr "erro en OCIHandleAlloc() HTYPE_ERROR"

#: providers/gda-oracle-server/gda-oracle-connection.c:379
#, c-format
msgid "Unknown ORACLE type %ld"
msgstr "Tipo %ld de ORACLE descoñecido"

#: providers/gda-oracle-server/gda-oracle-recordset.c:115
msgid "Unknown ORACLE data type in fill_field_values"
msgstr "Tipo de datos de ORACLE descoñecido en fill_field_values"

#: providers/gda-oracle-server/gda-oracle-recordset.c:150
msgid "recordset object being created without associated connection"
msgstr "o obxecto recordset estase a crear sen ningunha conexión asociada"

#: providers/gda-oracle-server/main-oracle.c:69
msgid "Could not register ORACLE provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor ORACLE"

#: providers/gda-postgres-server/gda-postgres-connection.c:888
#: providers/gda-postgres-server/gda-postgres-connection.c:1000
msgid "postgres_recset is NULL"
msgstr "postgres_recset é NULL"

#: providers/gda-postgres-server/main-postgres.c:69
msgid "Could not register PostgreSQL provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor PostgreSQL"

#: providers/gda-sybase-server/gda-sybase-command.c:115
msgid "Connection seems to be dead"
msgstr "Semella que a conexión está morta"

#: providers/gda-sybase-server/gda-sybase-command.c:135
msgid "Filling command structure failed\n"
msgstr "Fallou o enchido da estructura do comando\n"

#: providers/gda-sybase-server/gda-sybase-command.c:147
msgid "sending cmd failed"
msgstr "fallou o envío do comando"

#: providers/gda-sybase-server/gda-sybase-command.c:154
msgid "Allocating recordset failed"
msgstr "Fallou a asignación de memoria para o recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:164
msgid "Getting recordset userdata failed"
msgstr "Fallou a obtención dos datos de usuario do recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:222
msgid "Command execution finished, all results processed"
msgstr "Execución do comando rematada, procesados tódolos resultados"

#: providers/gda-sybase-server/gda-sybase-command.c:226
msgid "Command failed"
msgstr "Comando fallido"

#: providers/gda-sybase-server/gda-sybase-command.c:230
msgid "Execution of resultless command succeeded"
msgstr "Execución correcta dun comando sen resultado"

#: providers/gda-sybase-server/gda-sybase-command.c:246
#, c-format
msgid "ct_results() returned %x"
msgstr "ct_results() devolveu %x"

#: providers/gda-sybase-server/gda-sybase-command.c:256
msgid "No further results, command completly succeeded."
msgstr "Non hai máis resultados, o comando completouse correctamente."

#: providers/gda-sybase-server/gda-sybase-command.c:260
msgid "An error occured during command execution."
msgstr "Ocorreu un erro durante a execución do comando."

#: providers/gda-sybase-server/gda-sybase-command.c:267
msgid "Cmd execution returned unknown result, may be an error"
msgstr ""
"A execución do comando devolveu un resultado descoñecido, pode que sexa un "
"erro"

#: providers/gda-sybase-server/gda-sybase-command.c:280
msgid "No data returned, freeing recordset structure"
msgstr "Non se devolveu ningún dato, liberando a estructura recordset"

#: providers/gda-sybase-server/gda-sybase-command.c:311
#: providers/gda-sybase-server/gda-sybase-command.c:323
msgid "Freeing command structure failed"
msgstr "Fallou a liberación da estructura do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:149
msgid "Error allocating error structure"
msgstr "Erro asignando memoria para a estructura de erro"

#: providers/gda-sybase-server/gda-sybase-connection.c:157
msgid "Not enough memory for sybase_Connection"
msgstr "Non hai memoria dabondo para sybase_Connection"

#: providers/gda-sybase-server/gda-sybase-connection.c:422
#, c-format
msgid "schema_cols: table_name '%s'"
msgstr "schema_cols: table_name '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:438
msgid "Could not allocate command structure"
msgstr "Non foi posible asignar memoria para a estructura de comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:460
msgid "Could not allocate enough memory for query"
msgstr "Non foi posible asignar memoria dabondo para a consulta"

#: providers/gda-sybase-server/gda-sybase-connection.c:474
msgid "Could not get recordset in schema_cols"
msgstr "Non foi posible obte-lo recordset en schema_cols"

#: providers/gda-sybase-server/gda-sybase-connection.c:506
#: providers/gda-sybase-server/gda-sybase-connection.c:583
#, c-format
msgid "schema_tables: table name = '%s'"
msgstr "schema_tables: table name = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:514
#: providers/gda-sybase-server/gda-sybase-connection.c:591
#, c-format
msgid "schema_tables: table_schema = '%s'"
msgstr "schema_tables: table_schema = '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:519
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!\n"
msgstr "schema_procedures: proc_catalog = '%s' ¡SEN USAR!\n"

#: providers/gda-sybase-server/gda-sybase-connection.c:523
#, c-format
msgid "schema_procs: invalid constraint type %d"
msgstr "schema_procs: tipo de restricción %d non válida"

#: providers/gda-sybase-server/gda-sybase-connection.c:533
msgid "Could not allocate cmd structure"
msgstr "Non foi posible asignar memoria para a estructura cmd"

#: providers/gda-sybase-server/gda-sybase-connection.c:596
#, c-format
msgid "schema_procedures: proc_catalog = '%s' UNUSED!"
msgstr "schema_procedures: proc_catalog = '%s' ¡SEN USAR!"

#: providers/gda-sybase-server/gda-sybase-connection.c:600
#, c-format
msgid "schema_tables: invalid constraint type %d"
msgstr "schema_tables: tipo de restricción %d non válida"

#: providers/gda-sybase-server/gda-sybase-connection.c:661
#, c-format
msgid "Invalid or unsupported constraint type: %d"
msgstr "Tipo de restricción non válida ou non soportada: %d"

#: providers/gda-sybase-server/gda-sybase-connection.c:702
msgid "Could not allocate Gda_ServerCommand"
msgstr "Non foi posible asignar memoria para Gda_ServerCommand"

#: providers/gda-sybase-server/gda-sybase-connection.c:732
#: providers/gda-sybase-server/gda-sybase-connection.c:737
#, c-format
msgid "connection status: %x"
msgstr "estado da conexión: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:733
#, c-format
msgid "return code: %x"
msgstr "código devolvido: %x"

#: providers/gda-sybase-server/gda-sybase-connection.c:770
msgid "attempting reconnection"
msgstr "tentando a reconexión"

#: providers/gda-sybase-server/gda-sybase-connection.c:909
#, c-format
msgid "Reconnected to '%s'"
msgstr "Reonectado a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:911
#, c-format
msgid "Connected to %s"
msgstr "Conectado a %s"

#: providers/gda-sybase-server/gda-sybase-connection.c:919
#, c-format
msgid "Selected database '%s'"
msgstr "Seleccionada a base de datos '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:948
#, c-format
msgid "%s: Could not allocate cmd structure"
msgstr "%s: Non foi posible asignar memoria para a estructura do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:957
#, c-format
msgid "%s: Could not allocate memory for cmd string"
msgstr "%s: Non foi posible asignar memoria para a cadea do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:970
#, c-format
msgid "%s: ct_command failed"
msgstr "%s: ct_command fallou"

#: providers/gda-sybase-server/gda-sybase-connection.c:979
#, c-format
msgid "%s: sending command failed"
msgstr "%s: fallou o envío do comando"

#: providers/gda-sybase-server/gda-sybase-connection.c:990
#, c-format
msgid "%s: selecting database failed"
msgstr "%s: fallou a selección da base de datos"

#: providers/gda-sybase-server/gda-sybase-connection.c:1008
#, c-format
msgid "%s: use command failed"
msgstr "%s: o comando 'use' fallou"

#: providers/gda-sybase-server/gda-sybase-connection.c:1035
msgid "Could not allocate locale structure"
msgstr "Non foi posible asignar memoria para a estructura de locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1041
#, c-format
msgid "Could not set locale to '%s'"
msgstr "Non foi posible establece-lo locale a '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1047
#, c-format
msgid "Could not configure locale to be '%s'"
msgstr "Non foi posible configura-lo locale como '%s'"

#: providers/gda-sybase-server/gda-sybase-connection.c:1052
msgid "Could not drop locale structure"
msgstr "Non foi posible libera-la memoria da estructura de locale"

#: providers/gda-sybase-server/gda-sybase-connection.c:1065
#, c-format
msgid "Fatal error: %s\n"
msgstr "Erro moi grave: %s\n"

#: providers/gda-sybase-server/gda-sybase-error.c:65
#, c-format
msgid "gda_sybase_error_make(%p,%p,%p,%p)"
msgstr "gda_sybase_error_make(%p,%p,%p,%p)"

#: providers/gda-sybase-server/gda-sybase-error.c:74
#, c-format
msgid "No error flag ist set, though %s is called. This may be a bug."
msgstr ""
"Non hai ningunha marca de erro activa, pero chamouse a %s. Isto pode ser un "
"erro no código."

#: providers/gda-sybase-server/gda-sybase-error.c:138
msgid "cs_diag() initialized"
msgstr "cs_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:141
msgid "Could not initialize cs_diag()"
msgstr "Non foi posible inicializar cs_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:148
msgid "ct_diag() initialized"
msgstr "ct_diag() inicializado"

#: providers/gda-sybase-server/gda-sybase-error.c:151
msgid "Could not initialize ct_diag()"
msgstr "Non foi posible inicializar ct_diag()"

#: providers/gda-sybase-server/gda-sybase-error.c:185
#, c-format
msgid "%s could not find server implementation"
msgstr "%s non puido atopa-la implementación do servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:191
#, c-format
msgid "%s could not get cnc userdata"
msgstr "%s non puido obte-los datos de usuario de cnc"

#: providers/gda-sybase-server/gda-sybase-error.c:203
msgid "Could not get # of messages"
msgstr "Non foi posible obte-lo número de mensaxes"

#: providers/gda-sybase-server/gda-sybase-error.c:208
#, c-format
msgid "%d ctlib messages to fetch"
msgstr "%d mensaxes de ctlib para recibir"

#: providers/gda-sybase-server/gda-sybase-error.c:210
#, c-format
msgid "%d ctlib message to fetch"
msgstr "%d mensaxes de ctlib para recibir"

#: providers/gda-sybase-server/gda-sybase-error.c:246
msgid "error is nullpointer in gda_sybase_messages_handle"
msgstr "o erro é un punteiro a null en gda_sybase_messages_handle"

#: providers/gda-sybase-server/gda-sybase-error.c:271
#: providers/gda-sybase-server/gda-sybase-error.c:338
#: providers/gda-sybase-server/gda-sybase-error.c:381
msgid "Could not get # of server messages"
msgstr "Non foi posible obte-lo número de mensaxes de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:279
#, c-format
msgid "Fetching %d client messages"
msgstr "Recibindo %d mensaxes de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:281
#, c-format
msgid "Fetching %d client message"
msgstr "Recibindo %d mensaxe de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:290
#, c-format
msgid "Could not get client msg # %d"
msgstr "Non foi posible obte-la mensaxe de cliente %d"

#: providers/gda-sybase-server/gda-sybase-error.c:306
msgid "Could not get # of client messages"
msgstr "Non foi posible obte-lo número de mensaxes de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:314
msgid "Could not clear client messages"
msgstr "Non foi posible baleira-las mensaxes de cliente"

#: providers/gda-sybase-server/gda-sybase-error.c:346
#, c-format
msgid "Fetching %d server messages"
msgstr "Recibindo %d mensaxes de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:348
#, c-format
msgid "Fetching %d server message"
msgstr "Recibindo %d mensaxe de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:357
#, c-format
msgid "Could not get server msg # %d"
msgstr "Non foi posible recibi-la mensaxe de servidor %d"

#: providers/gda-sybase-server/gda-sybase-error.c:389
msgid "Could not clear server messages"
msgstr "Non foi posible baleira-las mensaxes de servidor"

#: providers/gda-sybase-server/gda-sybase-error.c:411
#: providers/gda-sybase-server/gda-sybase-error.c:447
msgid "Could not get # of cslib messages"
msgstr "Non foi posible obte-lo número de mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:419
#, c-format
msgid "Fetching %d cslib messages"
msgstr "Recibindo %d mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:421
#, fuzzy, c-format
msgid "Fetching %d cslib message"
msgstr "Recibindo %d mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:431
#, c-format
msgid "Could not get cslib msg # %d"
msgstr "Non foi posible recibi-la mensaxe de cslib %d"

#: providers/gda-sybase-server/gda-sybase-error.c:455
msgid "Could not clear cslib messages"
msgstr "Non foi posible baleira-las mensaxes de cslib"

#: providers/gda-sybase-server/gda-sybase-error.c:587
msgid "Server message:"
msgstr "Mensaxe de servidor:"

#: providers/gda-sybase-server/gda-sybase-error.c:596
msgid "Client message:"
msgstr "Mensaxe de cliente:"

#: providers/gda-sybase-server/gda-sybase-error.c:603
msgid "CS-Library error:"
msgstr "Erro de CS-Library:"

#: providers/gda-sybase-server/gda-sybase-recordset.c:152
#, c-format
msgid "fetch error at row %d"
msgstr "erro recibindo na ringleira %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:159
#, c-format
msgid "CS_END_DATA on row %d; canceling"
msgstr "CS_END_DATA na ringleira %d; cancelando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:168
#, c-format
msgid "%s: Error (%d)"
msgstr "%s: Erro (%d)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:343
msgid "Failed fetching # of columns"
msgstr "Fallou a recepción do número de columnas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:349
#, c-format
msgid "Counted %d columns"
msgstr "Contáronse %d columnas"

#: providers/gda-sybase-server/gda-sybase-recordset.c:356
#, c-format
msgid "%s could not allocate datafmt info"
msgstr "%s non puido asignar memoria para a información do formato de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:366
#, c-format
msgid "%s could not allocate data fields"
msgstr "%s non puido asignar memoria para os campos de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:384
#, c-format
msgid "ct_describe on col # %d"
msgstr "ct_describe na columna %d"

#: providers/gda-sybase-server/gda-sybase-recordset.c:391
#, c-format
msgid "ct_describe on col # %d failed"
msgstr "ct_describe na columna %d fallou"

#: providers/gda-sybase-server/gda-sybase-recordset.c:397
#, c-format
msgid "ct_describe(%d): '%s', %d (%s)"
msgstr "ct_describe(%d): '%s', %d (%s)"

#: providers/gda-sybase-server/gda-sybase-recordset.c:406
msgid "could not allocate data holder"
msgstr "non foi posible asignar memoria para o almacenador de datos"

#: providers/gda-sybase-server/gda-sybase-recordset.c:416
msgid "could not ct_bind data holder to column"
msgstr "non foi posible asignar con ct_bind o almacenador de datos á columna"

#: providers/gda-sybase-server/gda-sybase-recordset.c:423
msgid "illegal type detected, aborting"
msgstr "tipo ilegal detectado, abortando"

#: providers/gda-sybase-server/gda-sybase-recordset.c:429
msgid "could not allocate field"
msgstr "non foi posible asignar memoria para o campo"

#: providers/gda-sybase-server/main-sybase.c:74
msgid "Could not register gda-sybase provider implementation"
msgstr "Non foi posible rexistra-la implementación do fornecedor gda-sybase"

#: report/gda-report-engine.c:207
#, c-format
msgid "CORBA exception unloading report engine: %s"
msgstr "Excepción de CORBA o descargar o motor de informes: %s"

#: report/server/gda-report-server-main.c:41
#: report/server/gda-report-server-main.c:48
#, c-format
msgid "Received signal %d, shutting down."
msgstr "Recibiuse o sinal %d, saíndo."

#: report/server/gda-report-server-main.c:52
#, c-format
msgid "Received signal %d, shutting down cleanly"
msgstr "Recibiuse o sinal %d, saíndo limpamente"

#: report/server/gda-report-server-main.c:92
msgid "Failed to initialize OAF: please mail bug report to OAF maintainers"
msgstr ""
"Fallo ó inicializar OAF: por favor, envíe un informe do erro ós encargados "
"do OAF"

#: report/server/gda-report-server-main.c:101
msgid "Could not activate report engine"
msgstr "Non foi posible activa-lo motor de informes"

#: report/server/gda-report-server-main.c:116
msgid ""
"Another instance of the report engine is already registered with OAF. "
"Exiting..."
msgstr ""
"Xa hai rexistrada outra instancia do motor de informes con OAF. Saíndo..."

#: report/server/gda-report-server-main.c:119
msgid "Unknown error registering report engine with OAF. Exiting"
msgstr "Erro descoñecido rexistrando o motor de informes con OAF. Saíndo"

#: report/server/gda-report-server-main.c:123
#, c-format
msgid "Registered report engine with ID = %s"
msgstr "Rexistrado o motor de informes co ID = %s"

#: tools/gda-run.c:34
msgid "database name"
msgstr "nome da base de datos"

#: tools/gda-run.c:34
msgid "DATABASE"
msgstr "BASE-DE-DATOS"

#: tools/gda-run.c:35
msgid "user name"
msgstr "nome de usuario"

#: tools/gda-run.c:35
msgid "USER-NAME"
msgstr "NOME-DE-USUARIO"

#: tools/gda-run.c:36
msgid "password"
msgstr "contrasinal"

#: tools/gda-run.c:36
msgid "PASSWORD"
msgstr "CONTRASINAL"

#: tools/gda-run.c:37
msgid "file name"
msgstr "nome do ficheiro"

#: tools/gda-run.c:37
msgid "FILE-NAME"
msgstr "NOME-DO-FICHEIRO"

#: tools/gda-run.c:109
#, c-format
msgid "%s: misconfigured DSN entry"
msgstr "%s: entrada DSN mal configurada"

#: tools/gda-run.c:124
#, c-format
msgid "%s: data source '%s' not found\n"
msgstr "%s: fonte de datos '%s' non atopada\n"

#~ msgid "Fetchong %d cslib message"
#~ msgstr "Recibindo %d mensaxe de cslib"

#~ msgid "%s: not implemented"
#~ msgstr "%s: non implementado"
